mycpp/marksweep_gc_test.cc

```diff
@@ -35,6 +35,35 @@ TEST string_collection_test() {
   //
   // ASSERT(are_equal(test_str, StrFromC("foo")));
 
+  {
+    Str *test_str0 = nullptr;
+    Str *test_str1 = nullptr;
+    StackRoots _roots({&test_str0, &test_str1});
+
+    test_str0 = StrFromC("foo");
+    test_str1 = StrFromC("foo");
+
+    for (int i = 0; i < KiB(32); ++i)
+    {
+      if (i % 2)
```

Comment:
What is the purpose of this test?  It could use some comments

Reply:
Oh it was just some random junk I typed in to try and find bugs.  I'll delete it, it's not testing anything specific